common/management/commands/project_external_updates.py

```diff
@@ -72,7 +75,8 @@ def update_if_commit_after_project_updated_time(project, latest_commit_date_stri
 def add_commits_to_database(project, commits_to_ingest):
     from civictechprojects.models import ProjectCommit
     for commit_info in commits_to_ingest:
-        ProjectCommit.create(project, commit_info[0], 'master', commit_info[1])
+        branch = commit_info[2] if commit_info[2] is not None else 'master'
```

Comment:
It occurs to me this would be more readable if repo_display_name and commit name should also get their own variables here.

Reply:
Done